modules/apps/portal-crypto-hash/portal-crypto-hash-api/src/main/resources/content/Language.properties

```diff
@@ -0,0 +1,5 @@
+hash-pepper-storage-select=Hash Pepper Storage OSGi Target Filter Selector
```

Comment:
Check that added properties are either titles or sentences

Reply:
looks like they are